supplementary_style_guide/glossary_terms_conventions/general_conventions/j.adoc

```diff
@@ -99,8 +99,7 @@
 
 *Incorrect forms*:
 
-// TODO: Add links to "Red Hat Way" and "open source way".
-*See also*: Red Hat Way, open source way
+*See also*: link:https://www.redhat.com/en/open-source/red-hat-way[Red Hat Way], link:https://www.theopensourceway.org[open source way]
```

Comment:
I think what was meant here was to add an xref to the "Red Hat Way" entry in the SSG, can you switch to that? The "See also" field should pretty much just be for SSG entries. If we need to add a general "more info" link, then that should be in the description.

I don't see an entry for "open source way", so I'd just remove that one, or move it to the description.